### ISSUE TITLE
Unable to use the model - trying sentiment analysis

### ISSUE THREAD

**Issue by pk1762012** [facebookresearch/llama#501]
I downloaded the model. I gave the location where it is saved but it doesnt run. It asks for a config.json if running from transformers, and asking for model file when running from local. However, there are different parts of model downloaded - how can I give the model file name? Representative code below - I have tried multiple iterations with changing it to local,. etc. 

import torch
from transformers import AutoModelForSeq2SeqLM, AutoTokenizer

# Load model and tokenizer 
model_path = "~/LLAMA/llama/llama-2-13b"
model = AutoModelForSeq2SeqLM.from_pretrained(model_path)
tokenizer = AutoTokenizer.from_pretrained(model_path)

# Define sentiment analysis prompt
text = "I really enjoyed that movie!" 
prompt = f"Sentiment: {text}"

# Encode prompt
inputs = tokenizer(prompt, return_tensors="pt")

# Generate output 
outputs = model.generate(**inputs)
output = tokenizer.decode(outputs[0], skip_special_tokens=True)

# Print sentiment
print(output)

**Comment by pk1762012** [comment]
Yes I used a different code. Quantized version to solve it. We need to use 4bit or 8bit

**Comment by manojtudi** [comment]
Hey, did AutoModelForSeq2SeqLM work with llama-2-13b? It doesn't work for me.
